fdw/queries.sql.example

```diff
@@ -0,0 +1,12 @@
+drop foreign table similarity_search;
```

Comment:
I'm not sure what this file is for, but if it needs to be here, this first line should be:
`drop foreign table if exists similarity_search;`
and the use of `SMILES` below (line 10) should be removed.